composer.json

```
@@ -15,5 +15,10 @@
     },
     "autoload": {
         "psr-0": { "Doctrine_": "lib/" }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.3.x-dev"
```

I suggest to simplify it like the following:

```suggestion
            "dev-master": "1.3-dev"
```